GVFS/GVFS.Platform.Mac/MacFileSystemVirtualizer.cs

```diff
@@ -212,7 +212,8 @@ protected override bool TryStart(out string error)
 
             // Callbacks
             this.virtualizationInstance.OnEnumerateDirectory = this.OnEnumerateDirectory;
-            this.virtualizationInstance.OnGetFileStream = this.OnGetFileStream;
+            this.virtualizationInstance.OnGetFileStream = this.OnGetFileStream;
```

Comment:
All we need for the mirror provider is this line:

```
this.virtualizationInstance.OnLogError = this.OnLogError;
```

And an implementation of `private Result OnLogError(string errorMessage)` (it could just write the error to the console).

Reply:
I'll update